File: napa/DateCell.m
//
//  DateCell.m
//  napa
//
//  Created by Remy JOURDE on 21/06/2014.
//  Copyright (c) 2014 Remy Jourde. All rights reserved.
//

#import "DateCell.h"

@implementation DateCell

- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
{
    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
    if (self) {
        // Initialization code
    }
    return self;
}

- (void)awakeFromNib
{
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated
{
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
    if (selected)
    {
        self.selectedBackgroundView.hidden = YES;
        self.detailTextLabel.textColor = self.tintColor;
    }
    else
    {
        self.detailTextLabel.textColor = self.textLabel.textColor;
    }
    
}

@end

File: napa/DateCell.swift
//
//  DateCell.swift
//  napa
//
//  Created by Remy JOURDE on 11/10/2014.
//  Copyright (c) 2014 Remy Jourde. All rights reserved.
//

import UIKit

class DateCell: UITableViewCell {

    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        if selected {
            self.selectedBackgroundView.hidden = true
            self.detailTextLabel?.textColor = self.tintColor
        } else {
            if let textColor = self.textLabel?.textColor {
                self.detailTextLabel?.textColor = textColor
            }
        }

    }

}
